includes/class-sensei-course.php

```diff
@@ -3332,11 +3332,12 @@ public static function prerequisite_complete_message() {
 	 * @param WP_Post $course The Course.
 	 */
 	public function log_initial_publish_event( $course ) {
-		$product_id       = get_post_meta( $course->ID, '_course_woocommerce_product', true );
+		$product_ids = get_post_meta( $course->ID, '_course_woocommerce_product', false );
```

Comment:
Shouldn't this assignment be aligned with the one below? I'm surprised the linter didn't pick up on this...

Reply:
> I'm surprised the linter didn't pick up on this...

It's likely because of the blank line between them.